fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -389,6 +391,10 @@ public Response updateSparql(@ContentLocation final InputStream requestBodyStrea
      * application/octet-stream;qs=1001 is a workaround for JERSEY-2636, to ensure
      * requests without a Content-Type get routed here.
      *
+     * @author griffinj
```

Comment:
Method-level @author annotations probably not necessary.
...nor inline ticket reference below.